src/sections/node/channelsOutgoing.tsx

```diff
@@ -179,6 +180,66 @@ function ChannelsPage() {
       });
   };
 
+  const header = [
+    {
+      key: 'key',
+      name: 'key',
```

Comment:
Should we use the same name as in the incoming channels? (key is the same, but name is `#`)

Reply:
fixed